### DISCUSSION TITLE
Creating a sensor to detect sensor failures - is this possible?

### DISCUSSION THREAD

**Discussion by somiandras** [dagster-io/dagster#23902] [Q&A]
Sometimes I simply miss if one my sensors fail. It's clearly visible when I go to the sensors page, but it would be nice to create a sensor that notifies me if any other sensors fail due to some errors. Maybe there's some properties on `context.instance` that I could use? This way the problem could be reduced to a single point of failure.

**Edit:** to make it clear, I would want to monitor all of my sensors, not just a specific one. The response I got in #ask-ai only shows how to monitor a sensor via `context.instance.get_ticks()` by passing `origin_id` and `selector_id` obtained straight from the db for the sensor.

**Comment by gibsondan** [comment]
Hi @somiandras - unfortunately we don't have an efficient way to tail for sensor and schedule failures across all sensors - happy to take this as a feature request now. Right now you would have to iterate through each sensor (or the sensors that you care about) and fetch the most recent tick(s) for each one using the API that you mentioned.

**Reply by somiandras** [reply]
Thanks for confirming. Would be nice if Dagster supported this use case in some way.
